extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/ClientAssertionProvider.java

```diff
@@ -75,6 +75,11 @@ private ClientAssertion loadFromFileSystem() {
         if (Files.exists(bearerTokenPath)) {
             try {
                 String bearerToken = Files.readString(bearerTokenPath).trim();
+                if (bearerToken.isBlank()) {
```

Comment:
```suggestion
                if (bearerToken.isEmty()) {
```


It is trimmed in the line above

Reply:
oh yeah